apps/server/src/modules/provisioning/strategy/oidc/service/schulconnex-course-sync.service.spec.ts

```diff
@@ -181,7 +181,11 @@ describe(SchulconnexCourseSyncService.name, () => {
 			const setup = () => {
 				const studentUserId = new ObjectId().toHexString();
 				const teacherUserId = new ObjectId().toHexString();
-				const course: Course = courseFactory.build({ teacherIds: [teacherUserId], studentIds: [studentUserId] });
+				const course: Course = courseFactory.build({
+					teacherIds: [teacherUserId],
```

Comment:
Please add here at least two teacherIds

Reply:
We are testing here only the service which checks if at least 1 teacher is remaining. The part where it would differ if there were 2 or 1 teacher before is not part of this service, this happens in the strategy. It does not change the here when we add another teacher